src/combine.cpp

```diff
@@ -1538,12 +1538,16 @@ void do_symbol_check(Chunk *prev, Chunk *pc, Chunk *next)
 
    if (pc->Is(CT_AMP))
    {
+      Chunk *prevNext = prev->GetNext();
+
       if (prev->Is(CT_DELETE))
       {
          pc->SetType(CT_ADDR);
       }
       else if (  prev->Is(CT_TYPE)
-              || prev->Is(CT_QUALIFIER))
+              || prev->Is(CT_QUALIFIER)
+              || (  prevNext
```

Comment:
No need to test for "prev" to be not null. "GetNext()" is degined to never return a nullptr.

Reply:
done.